dist/google-keep-card.js

```diff
@@ -256,19 +247,12 @@ class GoogleKeepCard extends LitElement {
                     background-color: #e8eaed;
                     border-color: #e8eaed;
                 }
-
                 #googleKeepCard .checked {
                     color: #5f6368;
                 }
-                
                 #googleKeepCard .crossed {
                     text-decoration: line-through;
-                }
-                
-                #googleKeepCard .checked.uncrossed {
```

Comment:
That the exact google keep style. I don't mind particularly because I don't show the finished notes on my setup but if we want to be more close to google keep I recommend keeping it. Maybe we should add an option for `googleKeepStyle` that try to match perfectly all the small style?

Reply:
Ok, let's restore it